core/lib/mno_enterprise/platform_adapters/nex_adapter.rb

```diff
@@ -33,8 +36,61 @@ def fetch_assets
           copy_asset('s3://${MINIO_BUCKET}/assets/main-logo.png', logo_file)
         end
 
+        # @see MnoEnterprise::PlatformAdapters::Adapter#update_domain
+        def update_domain(domain_name)
+          domain = NexClient::Domain.new(cname: domain_name)
+          domain.relationships.origin = nex_app
+          domain.save
+
+          # Display errors if any
+          if domain.errors.any?
+            # display_record_errors(domain)
```

Comment:
I would advocate to return the domain anyway, and display the domain.errors in the controller.